cfnet/datasets.py

```diff
@@ -32,23 +31,52 @@ def _numpy_collate(batch):
   else:
     return np.array(batch)
 
-class NumpyLoader(DataLoader):
-  def __init__(self, dataset, batch_size=1,
+class DataLoader:
+    def __init__(self,dataset, batch_size=1,
```

Comment:
Please document the meaning of each argument

See how to write Jupyter Notebook documents:
https://birkhoffg.github.io/cfnet/tutorials/contribution.html#write-documentations-in-jupyter-notebook